src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -488,18 +488,16 @@ var DynamicTable = new Class({
             this.selectedRows.empty();
         },
 
-        selectRow : function (rowId) {
-            this.selectedRows.empty();
+        selectRow : function (rowId, clear) {
+            if (clear !== false)
```

Comment:
what about `clear === true` ? and below too.

Reply:
I used `clear !== false` so that existing instances of `selectRow(rowId)` don't need to be modified to include a second argument. I'm ok with either approach.

Reply:
Or how about add another method `selectRows()`?

Reply:
With a `selectRows()` method I would have to duplicate a lot of logic from `selectRow()`. I'd rather update all existing instances of `selectRow()` before that.

Reply:
>With a selectRows() method I would have to duplicate a lot of logic from selectRow()

That need not be if you moved all the work to `selectRows()` and `selectRow()` become merely:
```javascript
selectRow(rowId)
{
    selectRows([rowId]);
}
```

Reply:
That's a valid point. Though `selectRows()` would have enough branching logic (`deselectAll()`, `onSelectedRowChanged()`, `removeClass()`) that I think a separate `reselectRows()` function makes sense.

Reply:
>Though selectRows() would have enough branching logic

Well, I don't think that is a problem, `selectRow()` becomes trivial and one only need to read 1 func `selectRows()` instead of 2 (reselectRows + selectRow), it's much easier to understand.

Reply:
`reselectRows()` serves a very specific purpose with specific logic, whereas `selectRow()` is more of a general function. I'm happy with this current implementation, but I appreciate the different opinion.